lib/definitions.dart

```diff
@@ -0,0 +1,8 @@
+export 'package:sqlite_async/src/update_notification.dart';
```

Comment:
What's the purpose of this file / these exports? A large portion of these are already part of the sqlite_async.dart exports

Reply:
This was for testing in the early stages of splitting the different platform implementations. It's no longer needed and has been deleted. Thanks for spotting.